SdsClient.js

```diff
@@ -547,61 +533,68 @@ module.exports = {
       start,
       end
     ) {
-      return restCall({
+      return axios({
         url:
           this.url +
           this.streamsBase.format([tenantId, namespaceId]) +
           this.removeMultipleValuesBase.format([streamId, start, end]),
         method: 'DELETE',
         headers: this.getHeaders(),
-        gzip: true,
       });
     };
 
     // delete a type
     this.deleteType = function (tenantId, namespaceId, typeId) {
-      return restCall({
+      return axios({
         url:
           this.url +
           this.typesBase.format([tenantId, namespaceId]) +
           '/' +
           typeId,
         method: 'DELETE',
         headers: this.getHeaders(),
-        gzip: true,
       });
     };
 
     // delete a stream
     this.deleteStream = function (tenantId, namespaceId, streamId) {
-      return restCall({
+      return axios({
         url:
           this.url +
           this.streamsBase.format([tenantId, namespaceId]) +
           '/' +
           streamId,
         method: 'DELETE',
         headers: this.getHeaders(),
-        gzip: true,
       });
     };
 
     // delete a StreamView
     this.deleteStreamView = function (tenantId, namespaceId, streamViewId) {
-      return restCall({
+      return axios({
         url:
           this.url +
           this.streamViewsBase.format([tenantId, namespaceId]) +
           '/' +
           streamViewId,
         method: 'DELETE',
         headers: this.getHeaders(),
-        gzip: true,
       });
     };
 
+    this.gzipCompress = function (data, headers) {
+      if (
+        'Content-Encoding' in headers &&
+        headers['Content-Encoding'] == 'gzip'
```

Comment:
It is generally preferred to use `===` in JavaScript for equality checks. Might also want to consider using lowercase on the header, e.g.:
`headers['Content-Encoding'].toLowerCase() === 'gzip'`

Reply:
That's some interesting syntax. I have updated the line to your recommendation.